easybuild/scripts/rpath_args.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
```

Comment:
Hmm, ok, seems like it does work.

But still, this is irrelevant, since we call this script with `python rpath_args.py`, bypassing the shebang entirely...